config/locales/en.yml

```diff
@@ -846,8 +846,9 @@ en:
           remove: "Remove disbursement"
       expenses:
         textarea_additional_info:
-          additional_info_heading: Additional information (optional)
-          additional_info_label: Provide any information that will help in the assessment of your expenses
+          additional_info_heading: Additional information
+          agfs_additional_info_label: 'Provide any information that will help in the assessment of your expenses. For example, if you claim the private mileage rate on travel expenses, we need to know why.'
```

Comment:
not a blocker: for multi-line translations can use | (pipe) and single quotes are default:
```
   agfs_additional_info_label: |
     Provide any information that will help in the assessment of your expenses. For 
     example, if you claim the private mileage rate on travel expenses, we need to 
     know why.
```